encryptio.py

```diff
@@ -1,9 +1,55 @@
 #! /usr/bin/env python3
 
+import argparse
+from enum import Enum
+import os
+from pathlib import Path
+
 authorSignature = 'encryptio.py - Published by Moritz Nentwig and Laura Tzigiannis\n'
 authorSignature += '---------------------------------------------------------------'
 
 ## print header first 
 print("")
 print(authorSignature)
+print("")
+
+## possible encryption algorithms
+class EncryptionAlgorithms(Enum): 
+    aes='AES'
+    des='DES'
+    trippleDes='3DES'
```

Comment:
kein CamelCase 🐫

Reply:
Class names in python are in written CamelCase. The first letter is also uppercase. 

The entries of the enum have to be written uppercase.